CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 - Upgrade cilium-app to v0.23.0 in order to make Cilium ENI mode for CAPA usable (adds subnet and security group selection filters)
+- containerd: Hash configuration. ([#158](https://github.com/giantswarm/cluster/pull/158))
```

Comment:
```suggestion
- containerd: Append hash to configuration secret name so that nodes roll on changes ([#158](https://github.com/giantswarm/cluster/pull/158))
```

Reply:
Ah, yeah, you're right. Since there are multiple files to change, I think I'll add a general description and bullet points for each file later